.circleci/real_config.yml

```diff
@@ -1706,6 +1706,7 @@ jobs:
           root: .
           paths:
             - master/coverage.out
+            - master/internal/mocks
```

Comment:
"go test -cover now prints coverage summaries for covered packages that do not have their own test files. Prior to Go 1.22 a go test -cover run for such a package would report"

This change prevents our coverage step to fail since the new Go version requires our mocks be present.